src/main.ts

```diff
@@ -45,6 +82,11 @@ async function run(): Promise<void> {
     // Get the GitHub OIDC token.
     const githubOIDCToken = await core.getIDToken(audience);
 
+    if (tokenFormat === 'application_credentials') {
```

Comment:
I don't think "application_credentials" is a token format. This doesn't really make sense.